app/guid-node/metadata/-components/metadata-select/template.hbs

```diff
@@ -1,16 +1,16 @@
 <div local-class='metadata-select-container {{if this.isMobile 'mobile'}}'>
     <div local-class='title' data-test-select-title>
-        {{ this.template.schemaName }}
+        {{ @template.schemaName }}
     </div>
     <div local-class='description' data-test-select-description>
-        {{ this.template.template.description}}
+        {{ @template.template.description}}
     </div>
     <div local-class='action'>
         <Button
             data-test-select-button
             aria-label= {{ t 'metadata.add-flow-select' }}
             data-analytics-name='Select metadata'
-            {{on 'click' this.selectTemplate}}
+            {{on 'click' (fn @selectTemplate @template.cedarId) }}
```

Comment:
```suggestion
            {{on 'click' (fn @selectTemplate @template) }}
```

Reply:
But the template instance is already passed to this component as an argument. If we do this then we don't need to find the template by `cedarId` in `selectTemplate` action, which is a less direct approach than setting the `@template` argument passed into this component onto the `selectedTemplate` in the route controller. Since all objects are passed by reference, the `@template` here is the same object as the template instance contained in the route's `model.templates`. Might as well just rely on Ember/JS internal pointer reference instead of relying on the `cedarId` to make sure we are setting the correct `selectedTemplate`.